### DISCUSSION TITLE
Is there a way to turn off the Menu component's autocomplete ("Letter Navigation")?

### DISCUSSION THREAD

**Discussion by antondolinsky** [chakra-ui/chakra-ui#3149] [Q&A]
I want to implement my own autocomplete but the Letter Navigation feature (https://chakra-ui.com/docs/overlay/menu#letter-navigation) interferes with it. Is there a way to turn it off? I don't see any props that control it, but maybe there is some other way.

**Comment by solncebro** [comment]
> I want to implement my own autocomplete but the Letter Navigation feature (https://chakra-ui.com/docs/overlay/menu#letter-navigation) interferes with it. Is there a way to turn it off? I don't see any props that control it, but maybe there is some other way.

Did you find the way how to resolve this problem?? I'm trying make autocomplete based on Input and Menu components, but it is lose focus every time after backspacing and get variant. For example, my menu have a item "ololo", when I typed "olot, I've got empty list, but when I backspace one symbol to "olo" I've got item "ololo" (expectable), but lose focus (unexpecable). I don't know how it fix it.

**Reply by pedrobernardina** [reply]
Using `Box` instead of `MenuItem` seems to do the trick. Using `as={Box}` at MenuItem does not work. I don't know if there are any side effects.

**Reply by daliudzius** [reply]
Using `Box `doesn't carry with it the focus and arrow key functionality that `MenuItem `did. 
@solncebro Did you end up finding a solution to this, or did you write a custom component instead?

**Reply by solncebro** [reply]
> Using `Box `doesn't carry with it the focus and arrow key functionality that `MenuItem `did. @solncebro Did you end up finding a solution to this, or did you write a custom component instead?

I wrote a temporary crutch for it - just callback focus at the onBlur function by event.currentTarget.focus(). It is work, but it is a very dirty way, I think.

**Reply by daliudzius** [reply]
Ah gotcha, thank you. If I am getting too hung up on building out a new component, then I'll probably default to your fix. It's a bummer Chakra doesn't have either a `disableLetterNav` prop or a separate list item that retains the functionality of `MenuList`. 
In any case, the help was appreciated.

**Reply by daliudzius** [reply]
I was using Chakra `Menu` to build out a custom dropdown for search results. Although I was able to get this part to work, I found nice functionality and customization in `@chco-ui/autocomplete` which fit my needs even better. The main issue for me originally was finding a way to track the focused dropdown selection in order to show the Enter key and icon you see in the gif.

![screen_cap](https://user-images.githubusercontent.com/46355846/153931248-2e1cbf0f-68dd-4b26-9444-8fbc0920029d.gif)

**Comment by Exilibris** [comment]
Need a way to disable Letter Navigation since input placed in a MenuItem component doesn't accept A-z letters (I believe they're being captured by Menu)
